src/kibana/directives/aria_expanded.js

```diff
@@ -0,0 +1,23 @@
+define(function (require) {
+  require('modules')
+    .get('kibana')
+    .directive('ariaExpanded', function () {
+      return {
+        restrict: 'A',
+        link: function ($scope, $elem, attrs) {
+          $elem.bind('click', function () {
+            if ($elem[0].getAttribute('aria-expanded') === 'true') {
```

Comment:
In the future, yes, you should use `attr()` when interacting with jQuery elements.

Also, [as I mentioned above](#issuecomment-76280741), I don't think you even need this directive. I was just commenting so you can know more about standard practice.